tests/utils.py

```diff
@@ -49,3 +51,99 @@
     '_id': 'yes! yes! yes!',
     'count': 0
 }
+
+
+TEST_SCHEMA = update_schema(BASEXMLSCHEMA, {
+    "properties": {
+        "title1": "//dc:title/node()",
+        "title2": ["//dc:title/node()", lambda x: x.lower()],
+        "title3": ["//dc:title/node()", "//dc:title/node()", lambda x, y: x + y.lower()]
```

Comment:
Named selectors. Good for code-as-documentation.

On Mon, Mar 9, 2015 at 10:10 PM, Fabian von Feilitzsch <
notifications@github.com> wrote:

> In tests/utils.py
> <https://github.com/fabianvf/scrapi/pull/102#discussion_r26094446>:
>
> > @@ -49,3 +51,99 @@
> >      '_id': 'yes! yes! yes!',
> >      'count': 0
> >  }
> > +
> > +
> > +TEST_SCHEMA = update_schema(BASEXMLSCHEMA, {
> > +    "properties": {
> > +        "title1": "//dc:title/node()",
> > +        "title2": ["//dc:title/node()", lambda x: x.lower()],
> > +        "title3": ["//dc:title/node()", "//dc:title/node()", lambda x, y: x + y.lower()]
>
> what would the function of kwargs be?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fabianvf/scrapi/pull/102/files#r26094446>.
>